### ISSUE TITLE
What is the use of virtual platform?

### ISSUE THREAD

**Issue by rahulraveendran15-coder** [pulp-platform/pulpissimo#181]
Once the sdk is installed, the following commands can be executed in the sdk directory to use the virtual platform:

source sourceme.sh
source configs/platform-gvsoc.sh
Then tests can be compiled and run as for the RTL platform. When switching from one platform to another, it may be needed to regenerate the test configuration with this command:

make conf
More information is available in the documentation here: pulp-builder/install/doc/vp/index.html

Where can I get the manual the path given I am not able to find

Basically what is the purpose of this section?

**Comment by bluewww** [comment]
The virtual platform is a software-only model of the pulpissimo SoC (and also of other related SoCs). While a simulation of the hardware design is accurate it is also very very slow.
The virtual platform helps you develop software quicker by providing a more or less accurate software-model of pulpissimo. I would say running the virtual platform is about 1000x faster (maybe more).

Looks like the documentation path is out of date. You should find in in the `pulp-sdk` folder within the pulpissimo folder.
